File: Responder/Responder/Commands/QueryAppConfigCommand.cs
namespace Responder.AppConfig
{
    public class QueryAppConfigCommand : ICommand
    {
        #region Properties
        public CommandType Type { get; private set; }
        public int Length { get; set; }
        #endregion

        #region Constructors
        public QueryAppConfigCommand()
        {
            Type = CommandType.QueryAppConfig;
        }
        #endregion
    }
}

File: Imperium/src/com/imperium/Commands/QueryAppConfigCommand.java
package com.imperium.Commands;

public class QueryAppConfigCommand implements  ICommand {
    private CommandType type;
    private int length;

    public CommandType getType() {
        return type;
    }

    public int getLength() {
        return length;
    }

    public void setLength(int length) {
        this.length = length;
    }

    public QueryAppConfigCommand(){
        type = CommandType.QueryAppConfig;
    }
}
